batavia/types/Str.js

```diff
@@ -654,6 +654,14 @@ Str.prototype.__hash__ = function() {
     return new types.Int(h)
 }
 
+Str.prototype.capitalize = function() {
+    if(arguments.length === 0){
```

Comment:
I think for code style there are some spaces required around the brackets, ie. `if (...) {`